.vscode/launch.json

```diff
@@ -56,7 +56,8 @@
                 "program": "${workspaceFolder}/node_modules/jest/bin/jest.js"
             },
             "args": [
-                "${fileBasenameNoExtension}"
+                "--runTestsByPath",
+                "${relativeFileDirname}/${fileBasename}"
```

Comment:
Before, this cause running with "curve.ts" to run every test that includes "curve"